dao/mock_daos.go

```diff
@@ -555,8 +555,30 @@ func (m MockAuthenticationDao) GetById(id string) (*m.Authentication, error) {
 	return nil, util.NewErrNotFound("authentication")
 }
 
-func (m MockAuthenticationDao) ListForSource(sourceID int64, limit, offset int, filters []util.Filter) ([]m.Authentication, int64, error) {
-	panic("implement me")
+func (mAuth MockAuthenticationDao) ListForSource(sourceID int64, limit, offset int, filters []util.Filter) ([]m.Authentication, int64, error) {
+	sourceExists := false
+
+	for _, src := range fixtures.TestSourceData {
+		if src.ID == sourceID {
+			sourceExists = true
```

Comment:
I think you return already here and we don't need var `sourceExists`.

Reply:
🧐. I dont think so ... I need raise Not Found Error if source doesn't exist

Reply:
I see!